src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+
+
+import duke.commands.Command;
+import duke.exceptions.DukeInvalidDescriptionException;
+import duke.exceptions.DukeNoDescriptionException;
+import duke.tasks.TaskType;
+
+import java.util.Arrays;
+
+
+/**
+ * Parser for Duke.
+ * Parses user inputStrings.
+ */
+public class Parser {
+
+    /**
+     * Parse inputString to array by delimiter.
+     *
+     * @param inputString inputString line.
+     * @return parsed string array.
+     */
+    public static String[] parseLineToArray(String inputString) {
+        return inputString.split(" ");
+    }
+
+
+    /**
+     * Parses line and gets command.
+     *
+     * @param inputString input line.
+     * @return command.
+     */
+    public static Command getCommand(String inputString) {
+        String keyword = parseLineToArray(inputString)[0];
+
+        switch (keyword) {
+        case ("list"):
+            return Command.LIST;
+        case ("delete"):
+            return Command.DELETE;
+        case ("done"):
+            return Command.DONE;
+        case ("find"):
+            return Command.FIND;
+        case ("bye"):
+            return Command.TERMINATE;
+        case ("todo"):
```

Comment:
As the lack of a return statement in switch statements are very common, maybe we could add some //Fallthrough comments to show that it is intentional?